e2e/rbd.go

```diff
@@ -2768,6 +2772,10 @@ var _ = Describe("RBD", func() {
 					if err != nil {
 						e2elog.Failf("failed to delete PVC and application: %v", err)
 					}
+					err = deleteJournalInfoInPool(f, pvc, "replicapool")
+					if err != nil {
+						e2elog.Failf(" failed to delete omap data: %v", err)
+					}
```

Comment:
@Rakshith-R @Madhu-1 Moving deleteJournalInfoInPool() after deletePVCAndApp() causes CI failure

```
May  6 12:26:20.293: FAIL:  failed to delete omap data: failed to get pvc: persistentvolumeclaims "rbd-pvc" not found
```

see: https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/rest/organizations/jenkins/pipelines/mini-e2e_k8s-1.21/runs/3655/nodes/93/log/?start=0

Reply:
@pkalever can you remove this one I want to see what exactly is left behind? if we are not able to find what is the problem we can add it back. just want to give it a try with logs once.

Reply:
@pkalever do we have logs for this one?

Reply:
Do we have any issue opened for this one?

Reply:
Not yet Madhu, I will surely open one by EOD

Reply:
@Madhu-1 opened this one
https://github.com/ceph/ceph-csi/issues/3111